API_Nasa_Call.py

```diff
@@ -1 +1,89 @@
-# Calls Nasa's API to return a picture of the APOD and return the link to the image for now until Flask is setup.
\ No newline at end of file
+# Calls Nasa's API to return a picture of the APOD and return the link to the image for now until Flask is setup.
+
+'''This API ask a user to enter a specific date between beginning_date which is 1995-6-16 until today_date the current day
```

Comment:
Python docstrings typically use triple double quotes """

Reply:
I'll fix that. Thanks for the feedback.